build/Glyssen.proj

```diff
@@ -199,23 +199,47 @@
 		<Delete Files="$(RootDir)\output\RefTextUtilityInstallerinstaller\RefTextUtilityInstaller.msi" TreatErrorsAsWarnings="false" />
 
 	</Target>
+	
+	<!-- See https://wixtoolset.org/documentation/manual/v3/overview/insignia.html -->
+	<Target Name="SignCabs">
+		<PropertyGroup>
```

Comment:
We could save some duplicated code by putting the property group outside of the target. I think it would still work.

Reply:
Now moved into wixproj files, so they still need to be duplicated in each file, but I have now moved them outside their build targets in the wixproj files. In a normal build, the property will be set correctly based on the builld target (SignInstaller vs. SignIfPossible), so the properties in the wixproj files are just needed as fallbacks in case the installer is built in some other context (e.g., Visual Studio).